ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/EthProtocolConfiguration.java

```diff
@@ -14,169 +14,41 @@
  */
 package org.hyperledger.besu.ethereum.eth;
 
-import org.hyperledger.besu.util.number.PositiveNumber;
+import org.immutables.value.Value;
 
-import java.util.Objects;
+@Value.Immutable
```

Comment:
so does this annotation mean that everything in this class is immutable

Reply:
Yes, exactly